examples/word_language_model/README.md

```diff
@@ -0,0 +1,18 @@
+
+Mostly adapted from [here](https://github.com/pytorch/examples/tree/master/word_language_model).
```

Comment:
> what inferno contributed.

Yeah, as soon as we know for sure what that is and what it isn't. This is too much of a WIP to know for certain now.